lib/cryptography.js

```diff
@@ -14,7 +14,8 @@ const log = require('./utils/log')
 module.exports.decrypt = (options) => {
   try {
     const secret = options.secret || 'mySecret'
-    const inputFile = options.file || '.env.enc'
+    const inputFile = options.inputFile || '.env.enc'
+    const outputFilePath = options.outputFile || inputFile.replace('.enc', '')
```

Comment:
```suggestion
    const outputFilePath = options.outputFile
```

Use the path only if specified explicitly lets not do the replace thing